app/test_split_integer.py

```diff
@@ -19,3 +20,41 @@ def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
```

Comment:
This test function `test_should_add_zeros_when_value_is_less_than_number_of_parts` is currently a placeholder. Implement this test to verify that `split_integer` adds zeros when the value is less than the number of parts.